components/client/src/main/java/com/hotels/styx/client/connectionpool/ConnectionPools.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (C) 2013-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.styx.client.connectionpool;
+
+import com.hotels.styx.api.client.ConnectionPool;
+import com.hotels.styx.api.client.Origin;
+import com.hotels.styx.api.metrics.MetricRegistry;
+import com.hotels.styx.client.OriginStatsFactory;
+import com.hotels.styx.client.applications.BackendService;
+import com.hotels.styx.client.netty.connectionpool.NettyConnectionFactory;
+
+import static com.hotels.styx.client.connectionpool.ConnectionPoolSettings.defaultSettableConnectionPoolSettings;
+
+/**
+ * Helper routines for building connection pools with default settings.
+ */
+public final class ConnectionPools {
+    private ConnectionPools() {
+    }
+
+    public static ConnectionPool poolForOrigin(Origin origin) {
+        return new SimpleConnectionPool(origin, defaultSettableConnectionPoolSettings(), new NettyConnectionFactory.Builder().build());
+    }
+
+    public static ConnectionPool.Factory simplePoolFactory() {
+        return ConnectionPools::poolForOrigin;
+    }
+
+    public static ConnectionPool poolForOrigin(Origin origin, MetricRegistry metricRegistry) {
+        return new StatsReportingConnectionPool(
+                new SimpleConnectionPool(
+                        origin,
+                        defaultSettableConnectionPoolSettings(),
```

Comment:
Should we rename `defaultSettableConnectionPoolSettings` since "settable" is no longer in the class name?

Reply:
Makes sense. Will do.